addons/main.go

```diff
@@ -147,6 +148,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	//name of all the validating webhook configurations. the following need to be populatec with the names of the
+	//validating and webhooks configurations.
+	var vWhCfgNames []string
+	var mWhCfgNames []string
```

Comment:
This section needs to be done after all the webhooks are registered with the manager and we have all the names of the webhooks. we definetily don't want to hardcode the names.

Reply:
type ```populatec```